api-report/container-runtime.api.md

```diff
@@ -145,6 +145,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     get IFluidTokenProvider(): IFluidTokenProvider | undefined;
     get isDirty(): boolean;
     static load(context: IContainerContext, registryEntries: NamedFluidDataStoreRegistryEntries, requestHandler?: (request: IRequest, runtime: IContainerRuntime) => Promise<IResponse>, runtimeOptions?: IContainerRuntimeOptions, containerScope?: FluidObject, existing?: boolean): Promise<ContainerRuntime>;
+    static loadRuntime(context: IContainerContext, registryEntries: NamedFluidDataStoreRegistryEntries, existing: boolean, requestHandler?: (request: IRequest, runtime: IContainerRuntime) => Promise<IResponse>, runtimeOptions?: IContainerRuntimeOptions, containerScope?: FluidObject): Promise<ContainerRuntime>;
```

Comment:
Might be a good opportunity to switch to a single params object instead of all the params? Makes it easier to morph over time.  @anthony-murphy thoughts?

Reply:
sounds good. in general i prefer the pattern.  For cases like this where there is only one call per container instantiation using the object pattern is definitely worth it for the flexibility it gives us around compatibility. 

There are basically two cases where we might now want it:

1. called very frequently, as the object allocation cost could be high, but need to perf test the case individually to be sure
2. lots of call sites, as it can impact bundle size given the more verbose nature of the object pattern